### ISSUE TITLE
 D.D.// The Weeknd

### ISSUE THREAD

**Issue by AFcoder10** [surfbryce/beautiful-lyrics#911]
### Before you list your Songs:

- [X] IF you are reporting multiple songs, add ", ..." to the end of your title.
- [X] For every song you want to report, check Musixmatch or Apple Music and check to see if the Sync Type is better than what Beautiful Lyrics has.

### List Song Links

https://open.spotify.com/track/0WdJTI3UVvUMbLwPk5ObKI?si=7f11b13472ce437a

Hi, this song has lyrics but they are not time synced.
Pls add time synced lyrics for this songs.
Also, if you have karaoke synced lyrics you can put them, only if you have them.

Thank you!

**Comment by AFcoder10** [comment]
I dont use lyrics plus app anymore

**Comment by AFcoder10** [comment]
> The default built-in lyrics plus app provided by Spicetify has synced lyrics.

I dont use lyrics plus app anymore
There is time synced lyrics provided by spotify but not on beautiful lyrics

**Comment by DataM0del** [comment]
> > The default built-in lyrics plus app provided by Spicetify has synced lyrics.
> 
> I dont use lyrics plus app anymore There is time synced lyrics provided by spotify but not on beautiful lyrics

Yeah, it happens. There's some songs where Musixmatch or some other lyrics provider has synced lyrics but beautiful lyrics doesn't have them, probably because the client doesn't check for other services with synced lyrics (because theirs syncs line by line, and not by words or characters), but in that case, the solution is preferring beautiful lyrics' own lyrics provider over the others, so it would look like:
- Beautiful Lyrics (theirs)
- Musixmatch
- Lrclib
- Spotify
- Netease (sir, your syncing for the song "Running in the 90s" sucks, go in the corner now!!! */j*, but also Chyna)

**Comment by AFcoder10** [comment]
> > > > The default built-in lyrics plus app provided by Spicetify has synced lyrics.
> > > 
> > > 
> > > I dont use lyrics plus app anymore There is time synced lyrics provided by spotify but not on beautiful lyrics
> > 
> > 
> > Yeah, it happens. There's some songs where Musixmatch or some other lyrics provider has synced lyrics but beautiful lyrics doesn't have them, probably because the client doesn't check for other services with synced lyrics (because theirs syncs line by line, and not by words or characters), but in that case, the solution is preferring beautiful lyrics' own lyrics provider over the others, so it would look like:
> > ```
> > * Beautiful Lyrics (theirs)
> > 
> > * Musixmatch
> > 
> > * Lrclib
> > 
> > * Spotify
> > 
> > * Netease (sir, your syncing for the song "Running in the 90s" sucks, go in the corner now!!! _/j_, but also Chyna)
> > ```
> 
> the maintainer knows about this, he said that Musixmatch (which Spotify uses) will be added as a source soon. [#914 (comment)](https://github.com/surfbryce/beautiful-lyrics/issues/914#issuecomment-2391760727)

ok thank you!
